app/src/main/java/org/ole/planet/myplanet/ui/dashboard/DashboardFragment.java

```diff
@@ -208,28 +208,20 @@ public void onDestroy() {
 
     public void setCountText(int countText, Class c, View v) {
         if (c == RealmMyCourse.class) {
-            if (countText == 0) {
-                v.findViewById(R.id.count_course).setVisibility(View.INVISIBLE);
-            }
-            else {
-                ((TextView) v.findViewById(R.id.count_course)).setText(countText + "");
-            }
+            ((TextView) v.findViewById(R.id.count_course)).setText(countText + "");
```

Comment:
Initialized the same view twice instead create the object of TextView and use it like
`TextView tv_count_course = v.findViewById(R.id.count_course);
hideCountIfZero(tv_count_course, countText);`

Reply:
Good feedback